subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/TaskErrorOutputCaptureExecutionListener.kt

```diff
@@ -14,13 +14,14 @@ import org.jetbrains.kotlin.gradle.internal.KaptTask
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 internal class TaskErrorOutputCaptureExecutionListener(
-    private val logs: MutableMap<Path, StringBuilder>
+    private val logs: MutableMap<Path, StringBuilder>,
+    private val logger: CILogger
 ) : TaskExecutionListener, BuildAdapter() {
 
     override fun beforeExecute(task: Task) {
         when (task) {
             is Test -> {
-                task.ciLogger.debug("Add error listener to the task ${task.path}")
+                logger.debug("Add error listener to the task ${task.path}")
```

Comment:
fix unrelated bug - ci folder in the Teamcity build artifacts contains a lot of task-log files